### PR TITLE
*: adaptive analyze scan concurrency

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:

ref https://github.com/pingcap/docs/pull/17894

- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -971,7 +971,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`4`
-- 范围：`[1, 4294967295]`
+- 范围：`[0, 4294967295]`, v8.2 之前版本最小值为 `1`, 当设置为0时，会根据集群规模自适应并发度。
```
Comment (hfxsd) [inline system-variables.md:974]:
```suggestion
- 范围：`[0, 4294967295]`。在 v8.2.0 之前版本中，最小值为 `1`。当设置为 `0` 时，TiDB 会根据集群规模自适应并发度。
```

File: system-variables.md [hunk 2]
```diff
@@ -4871,7 +4871,7 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`1`
-- 范围：`[1, 4294967295]`，v7.5.0 及之前版本最大值为 `256`。
+- 范围：`[0, 4294967295]`，v7.5.0 及之前版本最大值为 `256`。 v8.2 之前版本最小值为 `1`, 当设置为0时，会根据集群规模自适应并发度。
```
Comment (hfxsd) [inline system-variables.md:4874]:
```suggestion
- 范围：`[0, 4294967295]`，v7.5.0 及之前版本最大值为 `256`。在 v8.2.0 之前版本中，最小值为 `1`。当设置为 `0` 时，TiDB 会根据集群规模自适应并发度。
```

**Review by songrijie — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -971,7 +971,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`4`
-- 范围：`[1, 4294967295]`
+- 范围：`[0, 4294967295]`。在 v8.2.0 之前版本中，最小值为 `1`。当设置为 `0` 时，TiDB 会根据集群规模自适应并发度。
```
Comment (lilin90) [inline system-variables.md:974]:
```suggestion
- 范围：`[0, 4294967295]`。在 v8.2.0 之前版本中，最小值为 `1`。当设置为 `0` 时，TiDB 会根据集群规模自适应调整并发度。
```

File: system-variables.md [hunk 4]
```diff
@@ -4871,7 +4871,7 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`1`
-- 范围：`[1, 4294967295]`，v7.5.0 及之前版本最大值为 `256`。
+- 范围：`[0, 4294967295]`，v7.5.0 及之前版本最大值为 `256`。在 v8.2.0 之前版本中，最小值为 `1`。当设置为 `0` 时，TiDB 会根据集群规模自适应并发度。
```
Comment (lilin90) [inline system-variables.md:4874]:
```suggestion
- 范围：`[0, 4294967295]`，在 v7.5.0 及之前版本中最大值为 `256`。在 v8.2.0 之前版本中，最小值为 `1`。当设置为 `0` 时，TiDB 会根据集群规模自适应调整并发度。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**